### PR TITLE
[event plane] update labeling for TPC

### PR DESCRIPTION
**Pull request by jikim1290**
update labeling for TPC

### CODE REVIEW

**Comment by strogolo** [comment]
@jikim1290 I fear that you have to do the same change also in the tasks that are currently using the Q-vector tables.

**Review by mfaggin — APPROVED** [review]
thanks @jikim1290 to me it looks fine for our single file using this tool

Just pinging also @stefanopolitano @gluparel @Libeiyu who use the code for their info (of course shout if I missed anything trivial)

**Review by mhemmer-cern — DISMISSED** [review]

**Review by skundu692 — APPROVED** [review]

**Review by skundu692 — APPROVED** [review]

**Comment by dsekihat** [comment]
Dear @jikim1290 , dileptonTable.h does not need to be modified. Thank you.

**Comment by jikim1290** [comment]
Hi @mfaggin @mhemmer-cern @skundu692 , I have to stress out that, in this change, I have also changed names of your all histograms including BPos, BNeg, and BTot, which might affects your post processing. It also changes your data model as well and I am asking this would be fine to be approved or let me know to go back. Thanks!

**Comment by dsekihat** [comment]
Hello, again,
histogram names in PWGEM/Dilepton/Utils/EventHistograms.h, dielectronQC.cxx, dimuonQC.cxx do not need to be modified, either. Thank you.

**Review by dsekihat — DISMISSED** [review]

**Comment by dsekihat** [comment]
Dear @jikim1290 , Thank your for changes. It looks good from PWG-EM.

**Review by skundu692 — APPROVED** [review]

**Comment by jikim1290** [comment]
Hi @ddobrigk @strogolo , Is it possible for this to be merged now? thanks

File: PWGDQ/DataModel/ReducedInfoTables.h
Comment (lucamicheletti93) [inline PWGDQ/DataModel/ReducedInfoTables.h:1]: Dear @jikim1290 , this change in the data model is problematic becasue breaks the processing of our DQ skimming tables. In particular you are changing the names of the variables in the reduced tables. If you wna to modify you need to preserve the old naming in the reduced tables (ReducedEventsQvectorCentr).

**Review by lucamicheletti93 — CHANGES_REQUESTED** [review]
Dear @jikim1290 , thanks a lot! See my comment here: https://github.com/AliceO2Group/O2Physics/pull/6861/files#r1686233799

File: PWGDQ/DataModel/ReducedInfoTables.h
Comment (lucamicheletti93) [reply PWGDQ/DataModel/ReducedInfoTables.h:1]: Thanks for implementing the changes!

**Review by lucamicheletti93 — APPROVED** [review]

**Comment by strogolo** [comment]
Dear @jikim1290,
this morning I had a chat with @lucamicheletti93 and we agree that you should fix the bugs in `PWGDQ/Core/VarManager.h` which are currently breaking the compilation. 
Furthermore, now there are also some conflicts with `PWGEM/Dilepton/TableProducer` that you should crosscheck and solve with the help of @dsekihat.
It should be important to solve all these aspects as soon as possible so to proceed with the uploading of the calibrations to CCDB.
Thanks and regards.

**Comment by jikim1290** [comment]
@strogolo I copied the mail to get your response:

If I understand correctly, you want to change labelling for all PWGs tasks as well.
I am one of people performing the service task, and updating codes in different PWGs is out of my permission.
Some people using Q-vectors want to keep the previous labelling, and I can not refuse their requests not to change them.
It means there is conflict between DPG and PWG, which should be resolved beforehand.
In this respect, I request you to announce that this is mandatory change that everyone should follow.
I know they want to have previous one for different reasons, which look reasonable to me, and I think this should be discussed with each other before the actual update.

In conclusions, one should state global agreement that no one can resist

**Comment by strogolo** [comment]
Dear @jikim1290,
I replied to your email and here we renew the invitation to act quickly to try to fix the bugs that are currently blocking the compilation of O2Physics (as reported by the tests in the PR) and resolve the conflicts reported in the PR because we think is time to converge on this and move forward to allow the analysers to proceed with their analyses.

File: Common/DataModel/Qvectors.h [hunk 1]
```diff
@@ -43,12 +43,12 @@ DECLARE_SOA_COLUMN(QvecFT0MReVec, qvecFT0MReVec, std::vector<float>);
...
-DECLARE_SOA_COLUMN(QvecBNegReVec, qvecBNegReVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBNegImVec, qvecBNegImVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBTotReVec, qvecBTotReVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBTotImVec, qvecBTotImVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCposReVec, qvecTPCposReVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCposImVec, qvecTPCposImVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCnegReVec, qvecTPCnegReVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCnegImVec, qvecTPCnegImVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCallReVec, qvecTPCallReVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCallImVec, qvecTPCallImVec, std::vector<float>);
```
Comment (vkucera) [inline Common/DataModel/Qvectors.h:51]: Please use proper camelCase for good readability. `Qvec`, `Cpos`, `Cneg`, `Call` are not words nor abbreviations.
```suggestion
DECLARE_SOA_COLUMN(QVecTpcPosReVec, qVecTpcPosReVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcPosImVec, qVecTpcPosImVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcNegReVec, qVecTpcNegReVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcNegImVec, qVecTpcNegImVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcAllReVec, qVecTpcAllReVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcAllImVec, qVecTpcAllImVec, std::vector<float>);
```

Comment (vkucera) [reply Common/DataModel/Qvectors.h:51]: This applied to all other name changes too.

**Review by vkucera — CHANGES_REQUESTED** [review]

**Comment by strogolo** [comment]
Dear @jikim1290,
this morning was approved and merged the PR done by @MaximVirta for the labelling following the strategy discussed and agreed with the analysis coordination. So I think we can close this PR since the change was already applied differently.

Thanks.

**Comment by strogolo** [comment]
Closed after PR #7199 was merged.

**Pull request closed without merging**